test/api/connectors/MockMtdIdLookupConnector.scala

```diff
@@ -18,11 +18,12 @@ package api.connectors
 
 import org.scalamock.handlers.CallHandler
 import org.scalamock.scalatest.MockFactory
+import org.scalatest.TestSuite
 import uk.gov.hmrc.http.HeaderCarrier
 
 import scala.concurrent.{ExecutionContext, Future}
 
-trait MockMtdIdLookupConnector extends MockFactory {
+trait MockMtdIdLookupConnector extends MockFactory with TestSuite {
```

Comment:
Why is TestSuite required across most of the Mock classes?

Reply:
This was because of the updated scalaMock dependency, which now requires TestSuite to be mixed in with MockFactory

Reply:
Your PR doesn't seem to change from 5.2.0 for ScalaMock and when I removed an example nothing changed. 

I would probably not go up to version 6.0.0 for now though as it's major version